.gitignore

```diff
@@ -13,3 +13,5 @@ coverage.html
 .glide
 .scBuildImage
 .init
+.generate*
+**/zz*.go
```

Comment:
> I would just prefer to not toss in the towel w/o trying.

No one's tossing in any towel.  Check it in for now, remove it in a follow up.  We have git.